aexpect/client.py

```diff
@@ -111,6 +111,8 @@ def __init__(self, command=None, a_id=None, auto_close=False, echo=False,
         self.a_id = a_id or data_factory.generate_random_string(8)
         self.log_file = None
         self.closed = False
+        # Use PYTHONENCODINGS or utf-8 (instead of ascii)
+        self.encoding = os.environ.get("PYTHONENCODING", "utf-8")
```

Comment:
Well ideally no one should need to know this. I mean over the years I saw only one complain related to encoding + from time to time I personally have some issues when running under my native locales. So in 99% of cases the ascii is fine, in 99.9999% of cases the utf-8 is right and the 0.0001% of users should dig deep to solve their issue...